src/main/java/io/kestra/plugin/azure/messaging/eventhubs/config/BlobContainerClientConfig.java

```diff
@@ -0,0 +1,44 @@
+package io.kestra.plugin.azure.messaging.eventhubs.config;
+
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.azure.client.AzureClientConfig;
+import io.kestra.plugin.azure.messaging.eventhubs.BlobContainerClientInterface;
+
+import java.util.Optional;
+
+/**
+ * Configuration for creating a new Azure EventHub Client.
+ */
+public interface BlobContainerClientConfig extends AzureClientConfig {
+
+    String containerName();
+
+    /**
+     * Default EventHubClientConfig
+     **/
+    record Default(
```

Comment:
Seems to not being used, you can remove it and also remove the interface that would be useless.